Description:
 - use Debian's jargs ard rhino libraries during build
 - don't embed jargs inside yui-compressor jar
Author: Michael Gilbert <michael.s.gilbert@gmail.com>
Author: Damien Raude-Morvan <drazzib@debian.org>
Last-Update: 2010-01-17
--- a/build.xml
+++ b/build.xml
@@ -36,8 +36,7 @@
 	           target="1.5"
                source="1.5">
             <classpath>
-                <pathelement location="${lib.dir}/jargs-1.0.jar"/>
-                <pathelement location="${lib.dir}/rhino-1.7R2.jar"/>
+                <pathelement location="/usr/share/java/jargs.jar"/>
             </classpath>
         </javac>
     </target>
@@ -46,13 +45,12 @@
         <mkdir dir="${build.dir}/jar"/>
         <!-- The order is important here. Rhino MUST be unjarred first!
              (some of our own classes will override the Rhino classes) -->
-        <unjar src="${lib.dir}/jargs-1.0.jar" dest="${build.dir}/jar"/>
-        <unjar src="${lib.dir}/rhino-1.7R2.jar" dest="${build.dir}/jar"/>
         <copy todir="${build.dir}/jar">
             <fileset dir="${build.dir}/classes" includes="**/*.class"/>
         </copy>
         <jar destfile="${build.dir}/${jar.name}" basedir="${build.dir}/jar">
             <manifest>
+                <attribute name="Class-Path" value="/usr/share/java/jargs.jar"/>
                 <attribute name="Main-Class" value="com.yahoo.platform.yui.compressor.Bootstrap"/>
             </manifest>
         </jar>
